### PR TITLE
(DOCSP-26976): Add link to C++ SDK object model page

### PR DESCRIPTION
**Pull request by krollins-mdb**
### Jira

- https://jira.mongodb.org/browse/DOCSP-26976

### Staged Changes

- [Create a Data Model](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-26976/sync/data-model/create-a-schema/#edit-your-realm-object-schema). Tab group at bottom of step 2 now includes C++.

### Reminder Checklist

You might need to also update some corresponding pages. Check if completed or N/A.

- [ n/a ] Create Jira ticket for corresponding docs-realm update(s), if any
- [ n/a ] Checked/updated Admin API
- [ n/a ] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### Animal in a hat

<image src="https://images-wixmp-ed30a86b8c4ca887773594c2.wixmp.com/f/10b82089-c0b9-4982-822c-64d544e8e0c5/dci1kyk-d267f960-c75f-4469-9c08-36ad60ecaf89.jpg/v1/fill/w_1280,h_1285,q_75,strp/cuttlefish_wearing_a_hat_by_drawrae_dci1kyk-fullview.jpg?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJ1cm46YXBwOjdlMGQxODg5ODIyNjQzNzNhNWYwZDQxNWVhMGQyNmUwIiwiaXNzIjoidXJuOmFwcDo3ZTBkMTg4OTgyMjY0MzczYTVmMGQ0MTVlYTBkMjZlMCIsIm9iaiI6W1t7ImhlaWdodCI6Ijw9MTI4NSIsInBhdGgiOiJcL2ZcLzEwYjgyMDg5LWMwYjktNDk4Mi04MjJjLTY0ZDU0NGU4ZTBjNVwvZGNpMWt5ay1kMjY3Zjk2MC1jNzVmLTQ0NjktOWMwOC0zNmFkNjBlY2FmODkuanBnIiwid2lkdGgiOiI8PTEyODAifV1dLCJhdWQiOlsidXJuOnNlcnZpY2U6aW1hZ2Uub3BlcmF0aW9ucyJdfQ.xajtbU0RY_quIqH82q14aINpSN5KlPiia3A7-mbUis4" width="400">

### CODE REVIEW

File: source/sync/data-model/create-a-schema.txt [hunk 1]
```diff
@@ -142,6 +142,10 @@ Create an App Services Schema from a Realm Object Model
 
             :ref:`Define a Realm Object Schema - Realm Kotlin SDK <kotlin-supported-types>`
 
+         .. tab::
```
Comment (dacharyc) [inline source/sync/data-model/create-a-schema.txt:145]: Bah. Unfortunately, adding C++ to the tab group is now causing it to be too wide to display all of the tabs. It has pushed React Native off the side of the tab list, and the tab list doesn't appear to scroll. (At least not for me in Chrome.)

We should probably change this from a tab list to a bullet list, instead, given this limitation. I'll ask about it in Slack to see if we have any work planned to support longer tab lists so we can go back to tabs for this in the future.

**Review by dacharyc — CHANGES_REQUESTED** [review]

File: source/sync/data-model/create-a-schema.txt [hunk 2]
```diff
@@ -271,44 +242,15 @@ Create a Realm Object Schema from an App Services Schema
...
+Refer to the Realm SDK-specific documentation to use the generated Realm Object Schema.
 
-      :ref:`Sync Data with the Realm Kotlin SDK <kotlin-sync-overview>`
+- :ref:`Realm Java SDK <java-sync-data>`
+- :ref:`Realm Swift SDK <ios-sync-changes-between-devices>`
+- :ref:`Realm Node.js SDK <node-sync-changes-between-devices>`
+- :ref:`Realm React Native SDK <react-native-realm-sync>`
+- :ref:`Realm .NET SDK <dotnet-sync-changes-between-devices>`
+- :ref:`Realm Flutter SDK <flutter-sync>`
+- :ref:`Realm Kotlin SDK <kotlin-sync-overview>`
```
Comment (cbush) [inline source/sync/data-model/create-a-schema.txt:253]: Should we alphabetize?
```
- :ref:`Realm .NET SDK <dotnet-sync-changes-between-devices>`
- :ref:`Realm Flutter SDK <flutter-sync>`
- :ref:`Realm Java SDK <java-sync-data>`
- :ref:`Realm Kotlin SDK <kotlin-sync-overview>`
- :ref:`Realm Node.js SDK <node-sync-changes-between-devices>`
- :ref:`Realm React Native SDK <react-native-realm-sync>`
- :ref:`Realm Swift SDK <ios-sync-changes-between-devices>`
```

**Review by cbush — APPROVED** [review]

File: source/sync/data-model/create-a-schema.txt (hunk 2 above)
Comment (krollins-mdb) [reply source/sync/data-model/create-a-schema.txt:253]: Ah, yeah, that probably makes sense. I'll update the order.

**Review by dacharyc — APPROVED** [review]
Awesome, thanks for rolling with the changes! I think this is definitely better.

**Pull request merged**